aws_jupyter_proxy/awsconfig.py

```diff
@@ -0,0 +1,19 @@
+from notebook.base.handlers import IPythonHandler, utcnow
+import json
+from botocore.session import get_session
+
+class AwsConfigHandler(IPythonHandler):
```

Comment:
Consider using the `ApiHandler` that is used for the other endpoint in this package as well. I believe that takes care of returning all points as JSON as well. 

https://github.com/aws/aws-jupyter-proxy/blob/d67537d641f94c91eb92ee0aa507b65a327a60b1/aws_jupyter_proxy/awsproxy.py#L63

Reply:
Sure, I will consolidate the handlers